assets/src/styles/blocks/Articles.scss

```diff
@@ -1,17 +1,28 @@
+.articles-block {
```

Comment:
These changes will eventually need to be implemented for other blocks too, but that requires some reviewing from the design team first

Reply:
@mleray can you circle back this one with the design team. If we don't have a plan currently to remove `.page-section-header` customization already, then the next best thing is to either do these changes [everywhere|https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/layout/_page-header.scss] or skip them for Articles block for now till we have a plan for all blocks.

Reply:
We were actually discussing it with @Inwerpsel, I agree that it would make sense to remove these changes from the current PR, since it's out of the ticket scope and it will be inconsistent with other blocks. I don't think we're ready to make the changes for all blocks, because the design team would need to review them first.

Reply:
[Removed](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/819/commits/b2ae665240a00c7f9ad8b8895b92dc09057fa7db) for now!